governance/votes/Executive Vote - March 11, 2022.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Template - [Executive Vote] Immunefi Bug Bounty Payouts, Curve stETH-ETH Onboarding, Rate Limited Flapper - March 11, 2022
```

Comment:
As a general rule of thumb, start with most significant items first when listing them.

Reply:
Fixed